docs/encrypted-innodb-tablespace-backups.md

```diff
@@ -353,7 +308,7 @@ In case the keyring has been rotated you’ll need to restore the keyring which
 
 ## Use the Key Management Interoperability Protocol (KMIP)
```

Comment:
Is this section necessary - we say it does on line 24.

Reply:
removed